packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -33,6 +33,12 @@ class ExportToCsvCommand extends Command {
       required: false,
       description: 'Name of the stack that needs to be created as csv filename.',
     }),
+    'stack-api-key': flags.string({
+      char: 'k',
+      multiple: false,
+      required: false,
+      description: 'Api key of the stack',
```

Comment:
Can we keep the description text the same as in another module like export/import?